netgrasp/netgrasp.py

```diff
@@ -66,114 +74,123 @@ def _load_debugger(self):
         self.logger.addHandler(self.debugger.handler)
 
     def _enable_debugger(self):
-        if self.daemonize:
+        if self.foreground:
```

Comment:
Half a dozen of one, six of the other: why are we renaming this flag / flipping its meaning? And again, it's unrelated to the MAC changes.